plugins/PostProcessingPlugin/scripts/Stretch.py

```diff
@@ -282,7 +283,7 @@ def wideCircle(self, orig_seq, modif_seq):
         dmin_tri is the minimum distance between two consecutive points
         of an acceptable triangle
         """
-        dmin_tri = self.line_width / 2.0
+        dmin_tri = 0.5
```

Comment:
This should not be hard coded to a fixed size. It makes more sense for this to depend on the print settings. Also, considering lines are often, say, 0.4mm, and this used to use 1/2 of that, bumping it up by 150% is a big jump!

Reply:
In fact, this value is more linked to CuraEngine computation precision than to nozzle size or line width. The post stretch algorithm tries to find in which direction the nozzle turns when extruding the material. If it turns right,  the algorithm moves the nozzle to the left. If it turns left, it moves it to the right, to compensate the tendency of the plastic to take short cuts.
If several successive points are close, CuraEngine tends to drop several points, to speed up the exchange with the instrument and, sometimes, rounds positions. The consequence is that the post stretch algorithm fails to find the correct direction of the turn, and the movement is not stable (erratic movements).
To have smooth movements, the algorithm looks 0,5mm ahead. It is like looking far away to estimate next turn instead of looking at the front tire of the vehicle ;-)

Reply:
Ah ok. We were misinterpreting what that variable does. Could possibly use a better name but thank you for your explanation.

In this case, we'll go ahead with testing this and then merging it.